rebasehelper/checker.py

```diff
@@ -40,6 +43,11 @@ def match(cls, cmd):
         """
         raise NotImplementedError()
 
+    @classmethod
+    def get_checker_output_dir_short(cls):
+        """Return short version of checker output directory"""
+        return os.path.join('rebase-helper-results/checkers', cls.CMD)
```

Comment:
Since you are already using `os.path.join()`, you shouldn't use `/` directly. Also there is `REBASE_HELPER_RESULTS_DIR` constant defined in `settings.py`.

```python
return os.path.join(REBASE_HELPER_RESULTS_DIR, 'checkers', cls.CMD)
```